os/check_out_of_memory/check_out_of_memory.py

```diff
@@ -64,7 +64,12 @@ def filter_entry_by_datetime(oom_list, hours_to_check):
         sys.exit(NAGIOS_EXIT_ERROR)
 
     oom_list = get_oom_entry()
-    matched_oom_list = filter_entry_by_datetime(oom_list, hours_to_check)
```

Comment:
@git001 , thanks for working on this.

I would suggest move the check of L68 to L72 into filter_entry_by_datetime function itself.